addons/crm/models/crm_lead.py

```diff
@@ -1853,7 +1868,7 @@ def _prepare_customer_values(self, partner_name, is_company=False, parent_id=Fal
             'is_company': is_company,
             'type': 'contact'
         }
-        if self.lang_id:
+        if self.lang_id.active:
```

Comment:
Mmmh ? Do we have issues with this (and in that case, isn't it a fix for stable ?) ?

Reply:
Thanks, I think I was about to introduce a bug because why not transfer the language of the lead to the partner even if that one is disabled. So I have reverted my change and modified my test.